cmd/storage-rpc-client.go

```diff
@@ -116,7 +116,8 @@ func newStorageRPC(endpoint Endpoint) StorageAPI {
 // Stringer interface compatible representation of network device.
 func (n *networkStorage) String() string {
 	// Remove the storage RPC path prefix, internal paths are meaningless.
-	serviceEndpoint := strings.TrimPrefix(n.rpcClient.ServiceEndpoint(), storageRPCPath)
+	serviceEndpoint := strings.TrimPrefix(n.rpcClient.ServiceEndpoint(),
+		path.Join(minioReservedBucketPath, storageRPCPath))
 	return n.rpcClient.ServerAddr() + ":" + serviceEndpoint
```

Comment:
Could you add a unit test to test `String()` function of network storage for a bunch of endpoint paths?